### PR TITLE
#18 - January blog post

### LINKED ISSUE #18: January release notes
Let's add release notes for January.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #18 
Added blog post

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 @genryxy please take a look

File: _posts/2021/01/2021-01-28-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+[v0.NN](https://example/link) was released on January NN. 
+We are happy to announce that minimal support for [Debian repository](https://wiki.debian.org/DebianRepository) was added to Artipie:
+you can [configure](https://github.com/artipie/artipie/tree/master/examples/debian) the repository 
+which [`apt-get`](https://en.wikipedia.org/wiki/APT_(software)) will be able to work with. 
+
+[Npm-adapter](https://github.com/artipie/npm-adapter) functionality was extended with the following commands support:
+- [`npm dist-tag`](https://github.com/artipie/npm-adapter/issues/155) for managing tags of package versions
+- [`npm deprecate`](https://github.com/artipie/npm-adapter/issues/156) for deprecating and un-deprecating the package
+- [`npm unpublish`](https://github.com/artipie/npm-adapter/issues/154) to un-publish entire package
```
Comment (genryxy) [inline _posts/2021/01/2021-01-28-release-notes.md:26]: @olenagerasimova thanks, one concern. Is it necessary hyphen in `un-deprecating` and `un-publish` words?

Comment (olenagerasimova) [reply _posts/2021/01/2021-01-28-release-notes.md:26]: @genryxy I do not think so as grammar checker says these worlds (not npm commands) should be written with hyphens.

**Comment by olenagerasimova** [comment]
@g4s8 please check, I've released Artipie and updated this post with the link

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**